generators/client/templates/react/src/main/webapp/app/shared/layout/header/header.tsx.ejs

```diff
@@ -9,7 +9,7 @@ import {
 import {
   FaHome, FaThList, FaUserPlus, FaUser, <%_ if (enableTranslation) { _%>FaFlag,<%_ } _%> FaHeart,
   FaList, FaTasks, FaDashboard, FaBook, FaWrench, FaSignIn, FaSignOut,
-  FaClockO, FaHddO,
+  FaClockO, FaEye, FaHddO,
```

Comment:
add the icon on new line with a // tslint:disable-next-line comment on top so that it doesnt cause failure when WebSocket is disabled